devtools/maven/src/main/java/io/quarkus/maven/CreateProjectMojo.java

```diff
@@ -245,7 +245,10 @@ private void askTheUserForMissingValues() throws MojoExecutionException {
                             .replace("_", ".") + ".HelloResource";
                     className = prompter.promptWithDefaultValue("Set the resource classname", defaultResourceName);
                     if (StringUtils.isBlank(path)) {
-                        path = prompter.promptWithDefaultValue("Set the resource path ", "/hello");
+                        String[] resourceClassName = StringUtils.splitByCharacterTypeCamelCase(
+                                className.substring(className.lastIndexOf(".")));
```

Comment:
It's a good idea. It would be better if it was a bit more solid though (think of a class in the default package for instance).

Reply:
yeah, I need to write a unit test to test all the possibilities and make it more solid.